includes/User.php

```diff
@@ -4903,19 +4903,15 @@ private function loadAttributes() {
 	}
 
 	private function compareAttributeValueFromService( $attributeName, $attributeValue ) {
-		if ( !array_key_exists( $attributeName, $this->mOptions ) ) {
-			$this->logAttributeMissing( $attributeName, $attributeValue );
-		} elseif ( $this->mOptions[$attributeName] !== $attributeValue  ) {
-			$this->logAttributeMismatch( $attributeName, $attributeValue );
+
+		// bio and coverPhoto are not used in MW
+		if ( $attributeName == "bio" || $attributeName == "coverPhoto" ) {
```

Comment:
Good point. For now we don't expect more attributes like this, and this logging will probably go away in the near future. If we add more attributes to this `if` statement though, we should split them into one place :)